nselib/http.lua

```diff
@@ -87,6 +87,12 @@
 --   end
 --   </code>
 -- * <code>no_cookie_overwrite</code>: Do not overwrite the cookies initially provided and when a duplicate cookie is received, ignore it.
+-- * <code>enable_cookies</code>: Cookies are accessible to other http calls
+-- The following example code demonstrates how the cookies can be passed and updated for every subsequent http calls.
+--
+-- <code> 
+-- http.get(host, port, {no_cookie_overwrite=false, cookies = options.cookies, enable_cookie = true})
```

Comment:
I got your point. But I am not very sure on how this is possible. Since the redirect calls are made from inside the library itself, its pretty simple to handle cookies in the next redirect request. Now, in a case when a cookie call is made from the script, we need to store the cookie jar from the previous request in the library somewhere for it to automatically parse in the next http call. I may be wrong here and need a bit more clarification on how to go about doing this.